regression/cbmc-incr-smt2/nondeterministic-int-assert/stdout-match.desc

```diff
@@ -0,0 +1,18 @@
+CORE
+test.c
+--outfile -
+Starting Bounded Model Checking
+\(set-option :produce-models true\)
+\(set-logic ALL\)
```

Comment:
:bulb: I suggest surrounding the SMT formula regexes here with ^ and $. That way they match the full line, which differentiates the output from the verbose debug output. (I recognise that the debug output is already switched off on this test).

Reply:
Added.